test_libs/pyspec/eth2spec/phase1/state_transition.py

```diff
@@ -0,0 +1,70 @@
+from . import spec
+
+from .spec import (
+    BeaconState,
+    BeaconBlock,
+    Slot,
+)
+
+from eth2spec.phase0.state_transition import (
```

Comment:
This whole file should be removed. (Mirroring what was done in #1018) It is actually unused as it stands and I forgot to remove it in my last commit. I will port whatever minimal functionality remains in it to other places tomorrow.

Reply:
Yes, I would actually suggest creating a phase 1 state transition .md file, that "integrates" the different parts of phase 1 into the state transition.